Create pull_request_template.md (#2756)
mozilla/blurts-server: .github/pull_request_template.md

<!-- The following is intended to be helpful to you. Feel free to remove anything that is not. -->

> # JIRA Ticket: 

Author: ```suggestion
# References: 
```

> MNTOR-

Author: ```suggestion
Jira: MNTOR-
Figma: 
```

<!-- When adding a new feature: -->

# Description

# Screenshot (if applicable)

Not applicable.

# How to test

# Checklist (Definition of Done)

> - [ ] l10n changes have been submitted to the l10n repository, if any.

Reviewer: For Relay, they have a separate repo/server I believe, and this line is more applicable to them.  For us, we might say something like below?
```suggestion
- [ ] Localization strings (if needed) have been added.
```

- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

> - [ ] I've added or updated relevant docs in the docs/ directory.

Reviewer: We don't have a docs directory yet in /src – should we add this, or try to reuse the one in root?

Author: Let me word it a little differently.. How about: 
`I've added or updated the relevant sections in readme and/or code comments`

Author: ```suggestion
- [ ] I've added or updated the relevant sections in readme and/or code comments
```
